CIP-0034/schema.cddl

```diff
@@ -0,0 +1,12 @@
+; defines a scheme to use for binary encoding for CBOR based APIs
+
+chain_id =
+{ 0 : bytes       ; Name
+  1 : network_id  ; NetworkId
```

Comment:
This would still expect the regular `0` / `1` ? Should we state that?

Just because on the binary [spec](https://github.com/input-output-hk/cardano-ledger/blob/17649bc09e4c47923f7ad103d337cc1b8e6d3078/eras/babbage/test-suite/cddl-files/babbage.cddl#L392)

When signing, it will still just expect to be just `0` / `1`.

Reply:
Not necessarily. I took this from the CIP-0034, where it's not 0/1 :)

Reply:
It's a 4-bit number in the actual Cardano spec. The binary spec is arguably wrong here since although 0 & 1 are the only two values used now, larger values up to 15 are reserved for future use